### PR TITLE
Fix TestS3Offload

### PR DESCRIPTION
**Pull request by sijie**
### Motivation

s3 mock service doesn't need credential. so we will fail to load credentials on s3 integration tests.
that is expected. However there is a regression when introducing GCS support in #2151:
https://github.com/apache/incubator-pulsar/pull/2151/files#diff-9160ffc9df7afcf42a8335f214393c6dR188

 ### Changes

Don't throw exception when failed to load aws credentials

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/offload/impl/BlobStoreManagedLedgerOffloader.java [hunk 1]
```diff
@@ -185,7 +185,6 @@ public static Credentials getCredentials(String driver, ServiceConfiguration con
             } catch (Exception e) {
                 // allowed, some mock s3 service not need credential
                 log.error("Exception when get credentials for s3 ", e);
```
Comment (ivankelly) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/offload/impl/BlobStoreManagedLedgerOffloader.java]: log.warn

Comment (sijie) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/offload/impl/BlobStoreManagedLedgerOffloader.java]: done

**Comment by sijie** [comment]
rerun java8 tests

**Comment by sijie** [comment]
rerun java8 tests

**Pull request merged**